waspc/examples/todoApp/ext/state.js

```diff
@@ -0,0 +1,3 @@
+const state = {}
```

Comment:
aha is this global local state? Is that on server or on the client? Maybe write that in the comments.